tests/test_data/spell_words.txt

```diff
@@ -167,4 +171,9 @@ Zach
 #------------------------------------
 #------ Stupid stuff to avoid false positive hits
 # Keep as short as possible, use \var{} etc. instead
-
+bd
```

Comment:
Keep as short as possible, use \var{} etc. instead for all entries on lines 174-179

Reply:
How do I use \var{} in this context? I know that how to use it in documentation but what about for this lintr spelling checks?

Reply:
For instance,
- look at the inline documentation of `extract_soil_CONUSSOIL`
- reading up the documentation, e.g., https://cran.r-project.org/doc/manuals/R-exts.html#Marking-text